pybliometrics/utils/startup.py

```diff
@@ -61,15 +69,43 @@ def check_default_paths(config: Type[ConfigParser], config_path: Path) -> None:
                 config.write(ouf)
 
 
-def create_cache_folders(config: Type[ConfigParser]) -> None:
+def check_keys_tokens() -> None:
+    """Auxiliary function to check if keys or tokens are set."""
+    keys = get_all_keys()
+    tokens = get_insttokens()
+    # 3 problematic cases
+    no_keys_no_tokens = True if not keys and not tokens else False
+    tokens_no_keys = True if tokens and not keys else False
+    keys_and_tokens = True if keys and tokens else False
+    keys_tokens_diff = len(keys) - len(tokens)
+
+    if no_keys_no_tokens:
+        raise ValueError('No API keys or InstTokens found. '
+                         'Please provide at least one key or token. '
+                         'For more information visit: '
+                         'https://pybliometrics.readthedocs.io/en/stable/configuration.html')
+    elif tokens_no_keys:
+        raise ValueError('API tokens found but not corresponding keys. '
+                             'Please provide the keys that correspond to the tokens. '
+                             'For more information visit: '
+                             'https://pybliometrics.readthedocs.io/en/stable/configuration.html')
+    elif keys_and_tokens:
+        if keys_tokens_diff < 0:
+            raise ValueError('More tokens than keys found. '
+                             'Please provide all the keys that correspond to the tokens. '
+                             'For more information visit: '
+                             'https://pybliometrics.readthedocs.io/en/stable/configuration.html')
```

Comment:
Stay consistent with the words please. Call it "API keys" and "InstTokens", and don't deviate from this.